rllib/algorithms/crr/crr.py

```diff
@@ -60,7 +58,9 @@ def __init__(self, algo_class=None):
         self.tau = 5e-3
 
         # overriding the trainer config default
-        self.num_workers = 0  # offline RL does not need rollout workers
+        self.num_workers = 4  # offline RL does not need rollout workers
+        self.offline_sampling = True
+        self.min_iter_time_s = 10.0
```

Comment:
Rename to `min_time_s_per_iteration`.

Reply:
`offline_sampling` is not a default AlgorithmConfig parameter. We need to add it to the `training` method, correct?